core/services/ocr2/plugins/ocr2keeper/evm/registry.go

```diff
@@ -565,29 +564,27 @@ func (r *EvmRegistry) checkUpkeeps(ctx context.Context, keys []types.UpkeepKey)
 	}
 
 	var (
-		err     error
-		results = make([]types.UpkeepResult, len(keys))
+		multiErr error
+		results  = make([]types.UpkeepResult, len(keys))
 	)
 
 	for i, req := range checkReqs {
 		if req.Error != nil {
-			if strings.Contains(req.Error.Error(), "reverted") {
```

Comment:
we do not expect these txs to revert and should treat those errors similar to the rest and append to multiErr